gobblin-service/src/test/java/org/apache/gobblin/service/modules/orchestration/DagManagerUtilsTest.java

```diff
@@ -109,15 +118,19 @@ public void testIsDagFinishedTwoNodes() throws URISyntaxException {
 
     setJobStatuses(dag, Arrays.asList(COMPLETE, PENDING));
     Assert.assertFalse(DagProcUtils.isDagFinished(dag));
+    Assert.assertEquals(TimingEvent.FlowTimings.FLOW_RUNNING, DagProcUtils.calcFlowStatus(dag));
```

Comment:
it took me a minute to realize you were repurposing the tests you just wrote for `isDagFinished` (which are very good BTW).  as a minor suggestion, consider renaming all of the test methods to reflect the dual purpose.  e.g. something like, `testFlowStatusPlusIsDagFinishedTwoNodes`